### PR TITLE
Remove list styling 

### PR DESCRIPTION
**Pull request by wheelsandcogs**
Removes list styling and uses default browser styles.
Also adds a default style for `<hr />`

### CODE REVIEW

File: assets/styles/_typography.scss [hunk 1]
```diff
@@ -8,6 +8,15 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
+ol li, ul li {
+  list-style-position: inside;
```
Comment (joefitter) [inline assets/styles/_typography.scss]: I thought we were binning this?

**Review by joefitter — DISMISSED** [review]

File: assets/styles/_typography.scss (hunk 1 above)
Comment (wheelsandcogs) [reply assets/styles/_typography.scss]: Binning it breaks all the li's everywhere (dashboard, breadcrumbs, etc).

File: assets/styles/components/_hint.scss [hunk 2]
```diff
@@ -1 +1,13 @@
...
+  ul, ol {
+    margin-left: $govuk-gutter-half;
+
+    li {
+      text-indent: -26px;
+      padding-left: 26px;
+      margin-bottom: 10px;
+    }
+  }
+}
```
Comment (joefitter) [inline assets/styles/components/_hint.scss]: these seem like "normal" lists? Why are we only applying these styles to lists in hints?

Comment (wheelsandcogs) [reply assets/styles/components/_hint.scss]: It adds the extra indentation that Affy wanted.

**Pull request merged**